drafts/concise_guide.ipynb

```diff
@@ -488,6 +602,86 @@
     "    return Response(f'The sum is {sess[\"sum\"]}.')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Handler functions can return the `HtmxResponseHeaders` object to set HTMX-specific response headers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@rt\n",
+    "def htmlredirect(foo:FastHTML): return HtmxResponseHeaders(location=\"http://example.org\")"
```

Comment:
You're not using `foo` here or referring to it, so either remove it or use it. Normally we'd just use the param `app` with no type IIRC so try that instead. Also see if you can merge this example with another one, especially since then it's clear you can return a tuple with headers along with fasttags etc.